docs/stability-platform-support.md

```diff
@@ -42,3 +42,68 @@ cases for for what `#[no_std]` might entail, so if you're interested in this
 we'd love to hear about your use case! Feel free to [open an
 issue](https://github.com/bytecodealliance/wasmtime/issues/new) on the
 `wasmtime` repository to discuss this.
+
+This is a common question we are asked, however, so to provide some more context
+on why Wasmtime is the way it is, here's some responses to frequent points
+raised about `#![no_std]`:
+
+* **What if my platform doesn't have `std`?** - For platforms without support
```

Comment:
This argument doesn't entirely cover other crates than Wasmtime in this repository, or in other repositories, such as `wasm-tools`. Would it make sense to include another question below, along the lines of "The crate I want to use without `std` works with small changes, why not accept those"?

After that one, yet another question could be "But I can't use `std`; what are my options?", where the answer could link to `build-std`, pointing out that it's Nightly-only for now, but progressing.